orcavault/models/dcl/sat_workflow_run.sql

```diff
@@ -0,0 +1,119 @@
+{{
+    config(
+        materialized='incremental',
+        incremental_strategy='append',
+        on_schema_change='fail'
+    )
+}}
+
+with ranked as (
+
+    select
+        sat.workflow_run_hk,
+        sat.record_source,
+        sat.workflow_name,
+        sat.workflow_version,
+        sat.state_status as workflow_run_status,
+        min(sat.state_timestamp) over (partition by sat.workflow_run_hk) as workflow_run_start,
+        max(sat.state_timestamp) over (partition by sat.workflow_run_hk) as workflow_run_end,
+        coalesce(sat.state_comment, cmt.comment) as workflow_run_comment,
```

Comment:
Not sure I fully get this, but I guess that's OK for now.

Reply:
No worries, can go through another time pair-sql-ing!